src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -24,11 +24,7 @@
 
 import com.google.common.collect.ImmutableMap;
 
-import io.appium.java_client.AppiumDriver;
-import io.appium.java_client.CommandExecutionHelper;
-import io.appium.java_client.FindsByAndroidUIAutomator;
-import io.appium.java_client.HasOnScreenKeyboard;
-import io.appium.java_client.LocksDevice;
+import io.appium.java_client.*;
```

Comment:
As far as I remember the linter does like star imports

Reply:
Fixed!